R/makeTreeSummarizedExperimentFromBiom.R

```diff
@@ -1,85 +1,14 @@
-#' Loading a biom file
-#'
-#' For convenience a few functions are available to convert data from a
-#' \sQuote{biom} file or object into a
-#' \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
-#'
-#' @param file biom file location
-#' 
-#' @param removeTaxaPrefixes \code{TRUE} or \code{FALSE}: Should
-#' taxonomic prefixes be removed? The prefixes is removed only from detected
-#' taxa columns meaning that \code{rankFromPrefix} should be enabled in the most cases.
-#' (default \code{removeTaxaPrefixes = FALSE})
-#' 
-#' @param rankFromPrefix \code{TRUE} or \code{FALSE}: If file does not have
-#' taxonomic ranks on feature table, should they be scraped from prefixes?
-#' (default \code{rankFromPrefix = FALSE})
-#' 
-#' @param remove.artifacts \code{TRUE} or \code{FALSE}: If file have
-#' some taxonomic character naming artifacts, should they be removed.
-#' (default \code{remove.artifacts = FALSE})
-#' 
-#' @param ... additional arguments 
-#'   \itemize{
-#'        \item{\code{patter}}{\code{character} value specifying artifacts
-#'        to be removed. If \code{patterns = "auto"}, special characters
-#'        are removed. (default: \code{pattern = "auto"})}
-#'    }
-#' 
-#' @return An object of class
```

Comment:
This documentation was little bit unclear (reflecting also the naming). But loadFromBiom is importer. So it cannot be added ton convert documentation.

Can you return the documentation that is related to loadFromBiom (I know that we have another PR open that renames importers)


The convert documentation is good

Reply:
loadFromBiom imports data from BIOM file. convert converts the class type